template/build/webpack.prod.conf.js

```diff
@@ -1,4 +1,3 @@
-var path = require('path')
```

Comment:
`path` is used in latest master: https://github.com/vuejs-templates/webpack/blob/master/template/build/webpack.prod.conf.js#L14-L15

Also, you should probably rebase your branch..